Final blog post voice assistant contest (#30866)
home-assistant/home-assistant.github.io: source/_posts/2024-01-17-voice-assistant-contest.markdown

---
layout: post
title: "Voice Assistant Contest - Let's build and win some prizes!"
description: "We are organizing a voice assistant contest between the 17th of January and the 10th of March. You can win some Home Assistant Green, some Home Assistant SkyConnect, and a chance to be on a livestream with us to talk about your creation."

> date: 2024-01-16 00:00:01

Reviewer: ```suggestion
date: 2024-01-17 00:00:01
```

date_formatted: "January 17, 2024"
author: Jean-Loïc Pouffier
comments: true
categories: Assist
og_image: /images/blog/2024-01-11-voice-assistant-contest/banner.png
---

<p><img src='/images/blog/2024-01-11-voice-assistant-contest/banner.png' class='no-shadow' /></p>

> TL;DR: We are organizing a voice assistant contest between the 17th of January and the 10th of March. You can win some [Home Assistant Green](https://www.home-assistant.io/green/), [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/), and a chance to be on a livestream with us to talk about your work.

Reviewer: ```suggestion
TL;DR: We are organizing a voice assistant contest between the 17th of January and the 10th of March. You can win some [Home Assistant Green](/green/), [Home Assistant SkyConnect](/skyconnect/), and a chance to be on a livestream with us to talk about your work.
```

<lite-youtube videoid="99lGuB4J-4o" videotitle="Voice Assistant Contest"></lite-youtube>

> 2023 was the [Year of the Voice](https://www.home-assistant.io/blog/2022/12/20/year-of-voice/). It was a yearly goal to **let users control Home Assistant in their own language**.

Reviewer: ```suggestion
2023 was the [Year of the Voice](/blog/2022/12/20/year-of-voice/). It was a yearly goal to **let users control Home Assistant in their own language**.
```

We built some amazing things throughout the year, from a very powerful intent recognizer specifically optimized to run on a small computer like a Raspberry Pi 4, all the way to custom wake words created by our community.

> In fact, we believe that what we built during the Year of the Voice allows almost anyone to build a voice assistant that embodies our Open Home values perfectly: Personalized, Private and Sustainable.

Reviewer: ```suggestion
In fact, we believe that what we built during the Year of the Voice allows almost anyone to build a voice assistant that embodies our Open Home values perfectly: Personalized, Private, and Sustainable.
```

Every component of our voice assistants can be heavily customized to fit your language, your style and your needs, fully local options exist for each of them, and finally you can retrofit voice assistant into anything, even a [1970s Walkie Talkie](https://www.youtube.com/shorts/HqYbhk6vq9E) found in a flea market

This is why, after spending a year building great things, we want to celebrate what our voice assistants can enable with the community: We’re launching a voice assistant contest, with great prizes to win!

> ## Contest Specifics

Reviewer: ```suggestion
## Contest specifics
```

The contest will be held between Today (the **17th of January**) and the **10th of March**.

We tried to be as inclusive as possible. The contest is not only about building voice assistant hardware, it is also about building voice experiences that can be used with any voice assistant (For example, [a blueprint that you can use to generate images from voice and cast them on your TV](https://my.home-assistant.io/redirect/blueprint_import?blueprint_url=https%3A%2F%2Fwww.home-assistant.io%2Fblueprints%2Fblog%2F2023-08%2Fcast_dall_e.yaml): “Show me a picture of an astronaut riding a unicorn!”)

An entry in the contest can take any form: A video, a blog post, a website, an event, etc.

The only requirement is for you to post on [this forum category](https://community.home-assistant.io/c/voice-assistant-contest/60), this is how you officially participate in the contest.

Winners will be announced on the **15th of March**.

A closing ceremony will take place on the **10th of April** in the form of a livestream. Winners will be invited!

<p><img src='/images/blog/2024-01-11-voice-assistant-contest/contest_timeline.png' class='no-shadow' /></p>

> ## Contest Categories

Reviewer: ```suggestion
## Contest categories
```

You can participate in the contest in four different categories.

### Most creative satellite ideas

We have all the ingredients to unleash creativity: Custom sentences, custom wake words, custom voices. This category is here to showcase your smartest ideas.
We will be judging the creativity, uniqueness, and coherency of the whole package.

Example: [Piitaya’s](https://github.com/piitaya) moving and talking droid.

<lite-youtube videoid="vQ7Hmeume9g" videotitle="Wake word demonstration on ESPHome-based 3D printed droid in Home Assistant | Year of the Voice"></lite-youtube>

> ### Best Starting Guides

Reviewer: ```suggestion
### Best starting guides
```

> Because our voice assistant can be heavily customized, starting can be daunting. This category is here to showcase the easiest ways to create a voice assistant for a novice. From ordering parts, flashing the firmware, integrating it into Home Assistant, and using it.

Reviewer: ```suggestion
Because our voice assistant can be heavily customized, starting can be daunting. This category showcases the easiest ways to create a voice assistant for a novice. From ordering parts, flashing the firmware, integrating it into Home Assistant, and using it.
```

> We will be judging how easy the guide is to follow, and how usable the final product is.

Reviewer: ```suggestion
We will judge how easy the guide is to follow and how usable the final product is.
```

<lite-youtube videoid="eTKgc0YDCwE" videotitle="Build the Wyoming Voice Satellite With ChatGPT and a Raspberry Pi To Replace Alexa & Google FOREVER!"></lite-youtube>

### Best events organized, best group effort

Building privacy-focused technology only makes sense if we can get people to use it. Go to a hackerspace, invite some people, and build some voice assistants together!
We will be judging the impact of the event (Size, outcome, topics discussed, etc)

### Best voice experiences

This category is here for people who want to participate in the contest but not necessarily build a hardware product. This is all about creating the best experience using a voice assistant on Home Assistant.
We support custom sentences, custom responses, and custom actions.
Go crazy, be creative, and show us how to unlock the potential of our voice assistants.
We will be judging how useful the experience is, how easy it is to integrate it, and of course, of fun it is.

Example: [Play any music on any speaker in your home from a blueprint.](https://blog.jlpouffier.fr/chatgpt-powered-music-search-engine-on-a-local-voice-assistant/)

> ## Contest Prizes

Reviewer: ```suggestion
## Contest prizes
```

For each category, the winner will receive:

> - a [Home Assistant Green](https://www.home-assistant.io/green/)
> - a [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/)

Reviewer: ```suggestion
- A [Home Assistant Green](/green/)
- A [Home Assistant SkyConnect](/skyconnect/)
```

> - A spot on the closing ceremony livestream to present his/her work, on the 10th of April

Reviewer: ```suggestion
- A spot on the closing ceremony livestream to present his/her work on the 10th of April
```

> Runners-up will receive a [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/).

Reviewer: ```suggestion
Runners-up will receive a [Home Assistant SkyConnect](/skyconnect/).
```

> The community will also get a chance to vote for a favorite entry, the choice of the community will also receive the same prizes as a category winner (A [Home Assistant Green](https://www.home-assistant.io/green/), A [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect/), a spot on the livestream on the 10th of April).

Reviewer: ```suggestion
The community will also get a chance to vote for a favorite entry. The choice of the community will also receive the same prizes as a category winner (A [Home Assistant Green](/green/), A [Home Assistant SkyConnect](/skyconnect/), a spot on the livestream on the 10th of April).
```
